build.gradle.kts

```diff
@@ -50,9 +50,6 @@ dependencies {
     }
 
     implementation("org.webjars:webjars-servlet-2.x:1.6")
-    implementation("org.webjars:jquery:3.6.1")
-    implementation("org.webjars:jquery-ui:1.13.2")
```

Comment:
one of these changes causes a browser console error, I assume the version of bootstrap relies on this (even if we aren't using that)

![image](https://user-images.githubusercontent.com/21194782/196123182-ffaa3dd6-4bf6-490b-bca4-15c37713ed4e.png)